extensions/panache/mongodb-panache/deployment/src/test/java/io/quarkus/mongodb/panache/DuplicateIdReactiveMongoEntityTest.java

```diff
@@ -0,0 +1,54 @@
+package io.quarkus.mongodb.panache;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.platform.launcher.Launcher;
+import org.junit.platform.launcher.LauncherDiscoveryRequest;
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
+import org.junit.platform.launcher.core.LauncherFactory;
+import org.junit.platform.launcher.listeners.SummaryGeneratingListener;
+import org.junit.platform.launcher.listeners.TestExecutionSummary;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DuplicateIdReactiveMongoEntityTest {
+    public static class DuplicateIdReactiveMongoEntityTestTest {
+        @RegisterExtension
+        static QuarkusUnitTest runner = new QuarkusUnitTest()
+                .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                        .addClasses(DuplicateIdReactiveMongoEntity.class))
+                .withConfigurationResource("application.properties");
```

Comment:
Hmm, I think we do not need to specify the configuration file since it is `application.properties` so it'll be picked by default.

Reply:
True: this was copied from an existing test but not needed.
Removed.